tests/phpunit/bootstrap.php

```diff
@@ -2,4 +2,13 @@
 
 declare(strict_types=1);
 
+use Symfony\Component\Dotenv\Dotenv;
+
 require __DIR__ . '/../../vendor/autoload.php';
+
+$dotenv = new Dotenv();
```

Comment:
Nestačí využívat toho že to pro dev nastaví docker-compose z `.env` file?
I když je pravda že díky tomuhle člověk nemusí restartovat kontejner když to spouští v něm a měnil ty nastavení v `.env`.

Trochu jsem s tím teď bojoval ale. Interpretuje to ten soubor trochu jinak než `docker-composer`.
Takže `PASSWORD=MUv9xitXWppZRE$HDzAfAUtAM23hoWYid` načetl jako `MUv9xitXWppZRE`.
Jde to ale obejít escapováním `PASSWORD=MUv9xitXWppZRE\$HDzAfAUtAM23hoWYid`.

Ještě předtím aby mi to vrátilo chybu jsem musel tady v bootstrapu nahodit errorhandler `BaseComponent::setEnvironment();`. To je ale chyba té třídy https://github.com/keboola/php-db-import/blob/76d3a3830a41332a8bbc3c8a6c30ff3c9595cc52/src/Snowflake/Connection.php#L78 connection která by se stejně měla časem vyhodit/předělat....

Reply:
To je myslím očekávané chování. Má se to chovat stejně jako v shellu. Takže 
```
HOST=myhost //myhost
USER=$HOST.myuser //myhost.myuser
```
Spíš docker-compose to asi automaticky escapuje.